core/lib/env_config/src/lib.rs

```diff
@@ -0,0 +1,42 @@
+use anyhow::Context as _;
+use serde::de::DeserializeOwned;
+
+mod alerts;
+mod api;
+mod chain;
+mod circuit_synthesizer;
+mod contract_verifier;
+mod contracts;
+mod database;
+mod eth_client;
+mod eth_sender;
+mod eth_watch;
+mod fetcher;
+mod fri_proof_compressor;
+mod fri_prover;
+mod fri_prover_gateway;
+mod fri_prover_group;
+mod fri_witness_generator;
+mod fri_witness_vector_generator;
+mod house_keeper;
+pub mod object_store;
+mod proof_data_handler;
+mod prover;
+mod prover_group;
+mod utils;
+mod witness_generator;
+
+#[cfg(test)]
+mod test_utils;
+
+pub trait FromEnv: Sized {
```

Comment:
I'll write doc-comments for new stuff later since I have a feeling that during the work on this project, some things will change. Probably it would be better to write doc-comments later than to do it now and then forget to fix them.